ansible/README.md

```diff
@@ -1 +1,33 @@
 # Ansible Collection - autoware.dev_env
+
+This collection contains the playbooks to set up the development environment for Autoware.
+
+## Set up a development environment
+
+### Ansible installation
+
+```bash
+# Remove apt installed ansible (In Ubuntu 22.04, ansible the version is old)
+sudo apt-get purge ansible
+
+# Install pipx
+sudo apt-get -y update
+sudo apt-get -y install python3-pip python3-venv
+
+python3 -m pip install --user pipx
```

Comment:
`pipx` is available on Ubuntu as a Debian package, is there anyting in the version from PyPI that we require? I'd prefer to use distro packages where possible.

https://packages.ubuntu.com/search?keywords=pipx

Reply:
I will test with this and report if it works 👍

Reply:
Updated with https://github.com/autowarefoundation/autoware/pull/4675/commits/a37bb05c318830edf1c325f8517f67d0bf8d258a

It turns out I already had `apt pipx` :sweat_smile: and `python3 -m pip install --user pipx` didn't do anything for me because I had the `apt pipx`.